# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../iniparser/iniparser-makefile.patch
# Copyright (C) 2010 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

---  iniparser3.0b/Makefile	2010-10-29 17:04:20.000000000 +0000
+++  iniparser3.0b/Makefile	2010-10-29 17:09:12.000000000 +0000
@@ -60,3 +60,7 @@
 	
 check:
 	@(cd test ; $(MAKE))
+
+install:
+	cp libiniparser.a libiniparser.so* /usr/lib; \
+	cp src/iniparser.h src/dictionary.h /usr/include
